codespell_lib/_codespell.py

```diff
@@ -742,4 +747,6 @@ def main(*args):
     if summary:
         print("\n-------8<-------\nSUMMARY:")
         print(summary)
-    return bad_count
+    if options.count:
+        print(bad_count, file=sys.stderr)
```

Comment:
Why don't we have some context to this anyway, people can still grep it out.

Reply:
Edit, I see this is exactly what flake8 does, seems kind of strange to me but it's good to be consistent with other stuff...